IDEs/eclipse_che/devfile.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: 1.0.0
+metadata:
+  name: OpenRefine_Che
+projects:
+  - name: OpenRefine
+    source:
+      location: 'https://github.com/OpenRefine/OpenRefine.git'
+      type: github
+      branch: master
+attributes:
+  persistVolumes: 'false'
+components:
+  - id: redhat/java11/latest
+    type: chePlugin
+  - mountSources: true
+    memoryLimit: 1536M
+    type: dockerimage
+    volumes:
+      - name: m2
+        containerPath: /home/user/.m2
+    alias: maven
+    image: 'quay.io/eclipse/che-java11-maven:nightly'
+    env:
+      - value: ''
+        name: MAVEN_CONFIG
+      - value: >-
+          -XX:MaxRAMPercentage=50 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4
+          -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true
+          -Xms20m -Djava.security.egd=file:/dev/./urandom -Duser.home=/home/user
+        name: MAVEN_OPTS
+      - value: >-
+          -XX:MaxRAMPercentage=50 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4
+          -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true
+          -Xms20m -Djava.security.egd=file:/dev/./urandom
+        name: JAVA_OPTS
+      - value: >-
+          -XX:MaxRAMPercentage=50 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
+          -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4
+          -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true
+          -Xms20m -Djava.security.egd=file:/dev/./urandom
+        name: JAVA_TOOL_OPTIONS
+  - id: ms-vscode/vscode-github-pullrequest/latest
+    type: chePlugin
+
+commands:
+  - name: maven build
+    actions:
+      - workdir: '${CHE_PROJECTS_ROOT}/OpenRefine'
+        type: exec
+        command: mvn clean install
```

Comment:
@wetneb what would be the real commands we want to expose for `mvn` here in the `devfile.yaml`?  First step is just to ensure that the dev environment can build/test inside Che, then I can work on exposing the UI endpoint as well, which I kinda started in the `manual_devfile.yaml`.  I have 3 files here so that I don't lose track of testing (I hit my quota today on Eclipse SaaS with all the testing! lolol)

Reply:
I don't know, I haven't looked into Eclipse Che configuration - I am only aware of the existence of Eclipse Che, but have never tried to use it myself.

Reply:
Its OK, I found my answer... at the bottom of the `refine.bat` more or less.  I'll see if I can make progress on this today or tomorrow.
```
:gotMvnHome
set MVN_ACTION=""%ACTION%""
if ""%ACTION%"" == ""build"" set MVN_ACTION=compile test-compile dependency:build-classpath
if ""%ACTION%"" == ""test"" set MVN_ACTION=test dependency:build-classpath
if ""%ACTION%"" == ""server_test"" set MVN_ACTION=test -f main
if ""%ACTION%"" == ""extensions_test"" set MVN_ACTION=test -f extensions
call "%MAVEN_HOME%\bin\mvn.cmd" process-resources
call "%MAVEN_HOME%\bin\mvn.cmd" %MVN_ACTION%
goto end
```